br/backup-and-restore-faq.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Backup & Restore FAQ
+summary: Learn about Frequently Asked Questions (FAQ) and the solutions of BR.
+category: FAQ
+---
+
+# Backup & Restore FAQ
+
+This document lists the Frequently Asked Questions (FAQ) and the solutions about Backup & Restore (BR).
+
+If the problems are not covered in this document and cannot be resolved, you can raise questions in the [AskTUG](http://asktug.com) community.
```

Comment:
AskTUG does not have an English interface. I think it's not appropriate to direct English readers to AskTUG. Maybe it's better to put here a link to the BR repo so users can create issues.
@YuJuncen @toutdesuite What do you think?